packages/core/strapi/lib/commands/utils/data-transfer.js

```diff
@@ -284,7 +284,11 @@ const getDiffHandler = (engine, { force, action }) => {
 
     Object.entries(context.diffs).forEach(([uid, diffs]) => {
       for (const diff of diffs) {
-        const path = `${uid}.${diff.path.join('.')}`;
+        let path = `${uid}.${diff.path.join('.')}`;
+        if (path.endsWith('.')) {
+          // remove trailing periods
+          path = path.slice(0, -1);
```

Comment:
Does that work in node14 too?

Reply:
Does this happen when there is no diff?
If this is the case we could simply do
```js
const path = [uid].concat(diff.path).join('.');
```

Reply:
Oops, I misinterpreted where the extra dot was coming from. Yeah, this works better. Updated.